### ISSUE TITLE
Project Manager can't be started after the reinstall

### ISSUE THREAD

**Issue by makemefeelgr8** [o3de/o3de#16716]
**Describe the bug**
Project Manager fails to start. This is an error I'm getting:
`Failed to register engine`
`[ERROR] o3de.register default o3de object folder C:/... does not exist`

**Assets required**
You can download `o3de_installer_2305_1.exe` from the o3de website.

**Steps to reproduce**
1. Install the engine
2. Create a project
3. Uninstall the engine
4. Remove all leftovers, like 3rd party folder and project folder
5. Install the engine again
6. Start the engine

**Expected behavior**
Project Manager should start.

**Actual behavior**
Project Manager fails to start.

**Screenshots/Video**
![image](https://github.com/o3de/o3de/assets/39099842/20110742-418f-4b55-abfd-86b76d0399a5)

**Found in Branch**
no idea

**Commit ID from [o3de/o3de](https://github.com/o3de/o3de) Repository**
no idea

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows
 - Version: 10
 - CPU [e.g. Intel I9-9900k , Ryzen 5900x, ]
 - GPU [AMD 6800 XT, NVidia RTX 3090]
 - Memory [e.g. 16GB]

**Additional context**

**Comment by drakeredwind01** [comment]
suggest removing `needs-triage` `needs-sig` and adding `triage/accepted` and `sig-network` and maybe `sig-build`.

**Comment by lumberyard-employee-dm** [comment]
@makemefeelgr8 Do you have a `C:\Users\<YourUserName>\.o3de` directory?
It sounds like the `o3de_manifest.json` file in that directory is corrupted.

**Comment by makemefeelgr8** [comment]
@lumberyard-employee-dm Yes, for some reason the `.o3de` directory remained, even after I've uninstalled the engine. Looks like there's a bug in the uninstaller.
I removed the `.o3de` folder, installed the engine, and now it works. Here's the file - [o3de_manifest.json.txt](https://github.com/o3de/o3de/files/12677146/o3de_manifest.json.txt)
It did point to directories of the old installation that don't exist anymore.
